### PR TITLE
add warning about partial supersession

### PR DESCRIPTION
**Pull request by clairebehue**
Communicate on plugin partial supersession by deephub native capabilities.
(no changelog.md in this plugin)
(https://app.shortcut.com/dataiku/story/98406/deephub-deprecate-superseded-plugins)

### CODE REVIEW

**Review by MayeulRousselet — APPROVED** [review]
LGTM

File: plugin.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
     "id": "object-detection-gpu",
-    "version": "0.1.6",
+    "version": "0.1.7",
     "meta": {
         "label": "Object detection in images (GPU)",
-        "description": "Detect objects in images using deep learning",
+        "description": "Detect objects in images using deep learning \n⚠️ Starting with DSS version 10 this plugin is partially superseded by the native [object detection capabilities](https://doc.dataiku.com/dss/latest/machine-learning/computer-vision/index.html).",
```
Comment (StanislasGuinel) [inline plugin.json]:
```suggestion
        "description": "Detect objects in images using deep learning.\n⚠️ Starting with DSS version 10 this plugin is partially superseded by the native [object detection capabilities](https://doc.dataiku.com/dss/latest/machine-learning/computer-vision/index.html).",
```

File: plugin.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 {
     "id": "object-detection-gpu",
-    "version": "0.1.6",
+    "version": "0.1.7",
     "meta": {
         "label": "Object detection in images (GPU)",
```
Comment (StanislasGuinel) [inline plugin.json:5]: should we add `partially superseded` in the plugin title ? I don't have a strong opinion on that

Comment (clairebehue) [reply plugin.json:5]: @MayeulRousselet told me not to do that 🤷‍♀️ i don't have a strong opinion on that either so i went with no change in the title

**Review by StanislasGuinel — APPROVED** [review]

**Pull request merged**